modules_basic/message-author.js

```diff
@@ -0,0 +1,39 @@
+var fs = require('fs')
+var Path = require('path')
+var h = require('../h')
+var when = require('@mmckegg/mutant/when')
+
+exports.needs = {
+  avatar_link: 'first',
+  avatar_image: 'first',
+  avatar_name: 'first',
+  timestamp: 'first'
+}
+
+exports.gives = {
+  message_author: true,
+  mcss: true
+}
+
+exports.create = function (api) {
+  return {
+    message_author,
+    mcss: () => fs.readFileSync(Path.join(__dirname, 'message-author.mcss'))
+  }
+
+  function message_author (msg, opts = {}) {
+    var { size = 1 } = opts
+    var { value } = msg
+    var { author } = value
+    return h('MessageAuthor', {
+      className: size === 0 ? '-mini' : ''
```

Comment:
not sure what 'change size to string' means ... you mean `size: mini` ? I think that might make sense